src/manifest/adaptation.ts

```diff
@@ -101,6 +101,9 @@ export default class Adaptation {
   /** Tells if the track is a trick mode track. */
   public isTrickModeTrack? : boolean;
 
+  /** Label of the adaptionSet */
+  public label?: string;
```

Comment:
Maybe this PR should be rebase to not include diffs by the #1105

Reply:
You're right, I'm not sure of how that PR catched that